.github/workflows/dolfinx-tests.yml

```
@@ -51,7 +51,7 @@ jobs:
         with:
           path: ./dolfinx
           repository: FEniCS/dolfinx
-          ref: main
+          ref: dokken/facet-expressions
```

```suggestion
          ref: main
```
Revert branch to main prior to merge